test/license_spec.js

```diff
@@ -0,0 +1,75 @@
+/*global contract, config, it, assert*/
+
+const License = require('Embark/contracts/License');
+
+let accounts;
+
+const zeroAddress = "0x0000000000000000000000000000000000000000";
+
+config({
+  contracts: {
+    License: {
+      args: [ zeroAddress, 1 ]
+    },
+  }
+}, (_err, web3_accounts) => {
+  accounts = web3_accounts
+});
+
+contract("License", function () {
+  it("should set recipient and price on instantiation", async function () {
+    const recipient = await License.methods.getRecipient().call();
+    const price = await License.methods.getPrice().call();
+    assert.strictEqual(parseInt(price, 10), 1);
+    assert.strictEqual(recipient, zeroAddress);
+  });
+
+
+  it("should not allow to buy license when price is incorrect", async function () {
+    try {
+      await License.methods.buy().send({value: 2, from: accounts[0]});
+    } catch(error) {
+      assert.strictEqual(error.message, "VM Exception while processing transaction: revert Value is not equal to expected price");
+    }
+  });
+
+  it("should allow to buy license", async function () {
+    let isLicenseOwner = await License.methods.isLicenseOwner(accounts[0]).call();
+    assert.strictEqual(isLicenseOwner, false);
+    await License.methods.buy().send({value: 1, from: accounts[0]});
+    isLicenseOwner = await License.methods.isLicenseOwner(accounts[0]).call();
+    assert.strictEqual(isLicenseOwner, true);
+  });
+
+  it("should not allow to buy license when the address already owns one", async function () {
+    try {
+      await License.methods.buy().send({value: 1, from: accounts[0]});
+    } catch(error) {
+      assert.strictEqual(error.message, "VM Exception while processing transaction: revert License already bought");
+    }
+  });
+
+  it("should allow set the price if owner", async function () {
+    try {
+      await License.methods.setPrice(10).send({from: accounts[1]});
+    } catch(error) {
+      assert.strictEqual(error.message, "VM Exception while processing transaction: revert");
+    }
```

Comment:
Could you split this test case in 2. One for it should and one for it should not.